README.md

```diff
@@ -34,6 +34,8 @@ Once imported, dhis2 selection filters can be called in as
     <ngx-dhis2-selection-filters
       (filterUpdate)="onFilterUpdateAction($event)"
       [dataSelections]="dataSelections"
+      [disablePeriodTypeSelection]="false"
```

Comment:
disablePeriodTypeSelection must be part of selectionFilterConfig, under periodFilterConfig